packages/rancher-istio-1.9/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-packageVersion: 1
```

Comment:
I'm a bit confused, do we need to bump Kiali within Istio, which is what caused the package bump?

Reply:
Yes, with every new kiali version we have to add it to our istio charts which results in an istio chart bump